### PR TITLE
DG: Fix assorted issues

### PR DESCRIPTION
**Pull request by wei2912**
See Google Docs for more details.

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -523,69 +430,73 @@ testers are expected to do more *exploratory* testing.
...
+      Expected: No person will be added. Error details shown in the status message. Status bar remains the same.
 
-   6. Test case: `add VALID_PREFIX/INVALID_INPUT`<br>
-      Expected: No person will be added. Error details shown in the status message. Status bar
-      remains the same.
+   6. Test case: `add VALID_PREFIX/INVALID_INPUT`
+      
+      Expected: No person will be added. Error details shown in the status message. Status bar remains the same.
 
 ### Editing a Person
```
Comment (lrnshk) [inline docs/DeveloperGuide.md:475]: You changed Person to person for Add command but not here.

**Review by lrnshk — CHANGES_REQUESTED** [review]
If you look at the rendered file, the bullet points for UC08 are spaced differently from the other use cases. So is NFRs and Glossary.

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (wei2912) [reply docs/DeveloperGuide.md:475]: Thanks, have amended this!

**Comment by wei2912** [comment]
> If you look at the rendered file, the bullet points for UC08 are spaced differently from the other use cases. So is NFRs and Glossary.

Thanks for pointing this out, have identified the issues and fixed in my latest commit.

**Review by lrnshk — APPROVED** [review]
LGTM

**Pull request merged**